src/resources/tls.cc

```diff
@@ -680,14 +680,14 @@ PRIMITIVE(set_session) {
     return tls_error(null, process, result);
   }
 
+  // Set the session and remember to always free the fake session
```

Comment:
Is it fake, or does mbedtls_ssl_set_session just take a copy?

Reply:
I just used the existing terminology from the deleted comment in line 688.